File: C/Palindrome.c
#include <stdio.h>

int main() {
    int num, temp, rev=0, digit;
    
    printf("Enter number: ");
    scanf("%d",&num);

    temp=num;

    while(num>0){
        digit=num%10;
        rev=rev*10+digit;
        num/=10;
    }

    if(temp==rev)
        printf("Palindrome");
    else
        printf("Not Palindrome");

    return 0;
}

File: Java/Palindrome.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        int temp = num;
        int rev = 0;
        while (temp > 0) {
            int digit = temp % 10;
            rev = rev * 10 + digit;
            temp = temp / 10;
        }
        if (rev == num) {
            System.out.println("Palindrome");
        } else {
            System.out.println("Not Palindrome");
        }
    }
}